pytorch_lightning/callbacks/finetuning.py

```diff
@@ -354,8 +334,8 @@ def __init__(
         initial_denom_lr: float = 10.0,
         train_bn: bool = True,
         verbose: bool = False,
-        round: int = 12,
-    ):
+        rounding: int = 12,
```

Comment:
No compatibility concerns about this rename?

Reply:
have checked and we do not user round as argument anywhere else...